### DISCUSSION TITLE
1.9.1 Release?

### DISCUSSION THREAD

**Discussion by rhapsodyv** [stm32duino/Arduino_Core_STM32#1347] [General]
We, at Marlin Firmware, use PlatformIO, that ships with the last release: 1.9.0. But it's almost 1 year behind the current master branch.

Can you do at least a minor release?

Thanks!

**Comment by fpistm** [comment]
Hi @rhapsodyv 
I know. I do my best to release the 2.0.0. We are closed to achieve the last stuff required and I've good hope to release next week.
In this context tt is not possible to do a minor release as several work commit in the master are related to the 2.0.0 and also several libraries has been updated and linked to 2.0.0.

**Reply by rhapsodyv** [reply]
Thank you!!
